pyproject.toml

```diff
@@ -16,13 +16,21 @@ extend_skip_glob = [
 [tool.pydocstyle]
 convention = "google"
 
+[tool.pylint.BASIC]
+good-names = [
+    "a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n",
```

Comment:
why is "o" not included?

Reply:
Good question :D Are there examples when `o` is used as a good name? I couldn't think of any when I was compiling the list

Reply:
I don't use one letter variable names very often. it's just that every other letter is there, so I was wondering about the omission.

Reply:
I could have added a comment but the ci would then have to run for another half day. I will merge this and add some explanation when I work on linting later.